packages/coq-serapi/coq-serapi.8.10.0+0.7.1/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer:   "e@x80.org"
+homepage:     "https://github.com/ejgallego/coq-serapi"
+bug-reports:  "https://github.com/ejgallego/coq-serapi/issues"
+dev-repo:     "git+https://github.com/ejgallego/coq-serapi.git"
+license:      "GPL-3.0-or-later"
+doc:          "https://ejgallego.github.io/coq-serapi/"
+
+synopsis:     "Serialization library and protocol for machine interaction with the Coq proof assistant"
+description:  """
+SerAPI is a library for machine-to-machine interaction with the
+Coq proof assistant, with particular emphasis on applications in IDEs,
+code analysis tools, and machine learning. SerAPI provides automatic
+serialization of Coq's internal OCaml datatypes from/to JSON or
+S-expressions (sexps).
+"""
+
+authors: [
+  "Emilio Jesús Gallego Arias"
+  "Karl Palmskog"
+  "Clément Pit-Claudel"
+  "Kaiyu Yang"
+]
+
+depends: [
+  "ocaml"               {           >= "4.07.0"            }
+  "coq"                 { !pinned & >= "8.10.0" & < "8.11" }
```

Comment:
```suggestion
  "coq"                 {           >= "8.10.0" & < "8.11" }
```
Correct me if I'm wrong here @ejgallego, but I don't believe `!pinned` is supposed to be used for released packages? At least other `opam` files for released packages do not.

Reply:
Indeed, we need to address this; we used this in CI and it seemed to work, but it was kinda strange. Do you mind opening an issue in the SerAPI repos?

I'll correct this PR manually.